api/v1alpha1/jobset_types.go

```diff
@@ -71,9 +71,15 @@ type ReplicatedJob struct {
 	Template batchv1.JobTemplateSpec `json:"template,omitempty"`
 	// Network defines the networking options for the job.
 	Network *Network `json:"network"`
+	// Replicas is the number of jobs that will be created from this ReplicatedJob's template.
+	// Pod hostnames for pods created by these jobs will be in the format:
+	// <jobsetName>-<jobName>-<jobIndex>-<podIndex>
+	Replicas *int `json:"replicas,omitempty"`
 }
 type Network struct {
 	// EnableDNSHostnames allows pods to be reached via their hostnames.
+	// Pods will be reachable using the fully qualified pod hostname, which is in
+	// the format: <jobsetName>-<jobName>-<jobIndex>-<podIndex>.<jobName>
```

Comment:
The subdomain will include the JobSet name as well, since it is the same as the job name, which we set to `<jobSet.name>-<spec.replicatedJob.name>-<job-index>` as you noted. 

So the final fully qualified pod hostname is: `<jobSet.name>-<spec.replicatedJob.name>-<job-index>-<pod-index>.<jobSet.name>-<spec.replicatedJob.name>-<job-index>`

Reply:
Yes, please add that